### DISCUSSION TITLE
Using bare ssh pubkeys

### DISCUSSION THREAD

**Discussion by mykhal** [FiloSottile/age#289] [General]
The ability to encrypt to ssh (even ssh-rsa) pubkeys is great!

Possibility to use just the b64-encoded "blob" (`AAA….`), without the initial key type text (like `ssh-rsa`), would be nice improvement, as these are redundant (my [microanalysis](https://github.com/mykhal/misc-crypto/tree/main/ssh-pubkeys) could be somewhat handy).

**Comment by FiloSottile** [comment]
They are indeed redundant, but considering them part of the recipient encoding has two advantages: authorized_keys files work out of the box with `-R`, and there is a visual indicator that an SSH key is in use. It also makes it easier to programmatically recognize recipient types from the prefix. Supporting both would be extra complexity, which we try to minimize as much as possible.
